schemas/rum/resource-schema.json

```diff
@@ -252,6 +252,12 @@
               "description": "trace identifier in decimal format",
               "pattern": "^[0-9]+$",
               "readOnly": true
+            },
+            "rule_psr": {
+              "type": "number",
+              "description": "sample rate in decimal format",
+              "pattern": "^[0-9]+$",
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏I guess we are not limited to integers, no?

Reply:
Looking at your change, I was probably not clear enough, I am thinking that we could provide values like `12,34` for example, so we should not limit that.

Reply:
It should be clearer using the [range](https://json-schema.org/understanding-json-schema/reference/numeric.html?highlight=number#range) keys mentioned by @BenoitZugmeyer